### ISSUE TITLE
[Question / Feature Request] Configurable pinned application update interval

### ISSUE THREAD

**Issue by kurac338** [linuxserver/Heimdall#642]
Hello,
I tried searching around for this, but did not find anything.

TL;DR: I am looking for a way to configure how often Heimdall will pull / refresh data from the dashboard applications.

Environment:
I use a RPI 4 (4 GB variant) as the dashboard, and currently have 11 pinned items. 7 of those items are enhanced applications that get auto-refreshed. 6 out of 7 apps are hosted on that same Pi, inside the same Docker stack. They are being queried through the Traefik reverse proxy for easy access in LAN and WAN.

Problem:
Once you land on the Heimdall dashboard, the poor Pi starts to spike the CPU usage and thermals.

Proposed solution:
1) Allow for the pooling interval to be a Docker-compose environment variable
2) Add a feature toggle to pool automatically or manually. Manual queries could be executed via a button on the dashboard?

**Comment by KodeStar** [comment]
The plan is to copy over some of the front end from heimdalljs, so you will be able to specify that apps are refreshed on demand (when you hover over a refresh icon will appear)
